roles/heat/tasks/main.yml

```diff
@@ -44,6 +45,17 @@
     - restart heat services
   # we want this to always be changed so that it can notify the service restart
 
+- name: setup heat domain and user
+  shell: . /root/stackrc; heat-keystone-setup-domain "{{ secrets.service_password }}" | grep stack_user_domain_id
+  register: heat_domain
+  failed_when: false
+
+- name: configure heat for stack domain
+  template: src=etc/heat/heat_stack_domain.conf} dest=/etc/heat/heat_stack_domain.conf mode=0644
+  when: heat_domain.rc == 0
```

Comment:
good question.   this is only needed while the keystone_role etc modules aren't ready.

this is an MVP implementation to make magnum usable ...  It's not meant to be production ready.  I should actually gate this bit off behind the magnum feature flag.